### PR TITLE
[Keys][P1] Implement the API Keys File Reader Component

### PR DESCRIPTION
**Pull request by deerhunter4**
[Keys][P1] Implement the API Keys File Reader Component

I created keys.py file. This component will be responsible for
handling user API keys stored in better_weather_keys.txt file.

### CODE REVIEW

File: keys.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    # read input file as a dictionary
+    keys_dict = {}
+    for line in keys_file:
+        key, value = line.strip().split(',')
+        keys_dict[key] = value
+
+    keys_file.close()
+
+    if 'wheatherapi.com' not in keys_dict or 'meteostat.net' not in keys_dict:
```
Comment (KrzysiekJodlowski) [inline keys.py:23]: `wheatherapi.com` -> `weatherapi.com`

File: keys.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        print("""The keys are missing in the better_weather_keys.txt file
+              or their names might be incorrect. The correct names are:
+              'wheatherapi.com', 'meteostat.net'""")
+        exit()
+
+    return keys_dict
```
Comment (KrzysiekJodlowski) [inline keys.py:29]: I'd suggest extracting this part into a separate function and calling it in line 9.

File: keys.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def check_keys():
+    keys_path = './better_weather_keys.txt'
```
Comment (KrzysiekJodlowski) [inline keys.py:5]: Please use [Python constants](https://realpython.com/python-constants/#what-constants-are) for such non-changeable variables.

**Review by KrzysiekJodlowski — CHANGES_REQUESTED** [review]
Apart from the comments just a `.pyc` reminder:)

File: keys.py (hunk 1 above)
Comment (deerhunter4) [reply keys.py:23]: done

File: keys.py (hunk 3 above)
Comment (deerhunter4) [reply keys.py:5]: Done.

File: keys.py (hunk 2 above)
Comment (deerhunter4) [reply keys.py:29]: Done. I created read_keys() function in keys.py, which is used by check_keys() function, as you suggested.

**Pull request closed without merging**